### PR TITLE
Added team 2 agent design section

### PR DESCRIPTION
**Pull request by yp717**
Will address any changes req first thing tomorrow morning bc of some personal family stuff I have to go deal with :/

### CODE REVIEW

**Review by catherinechaya — APPROVED** [review]
Looks Good! Great English Skills.

**Review by catherinechaya — APPROVED** [review]

File: 10_team2_agentdesign/index.tex [hunk 1]
```diff
@@ -1 +1,219 @@
...
+    \centering
+    \includegraphics[width=0.6\textwidth]{10_team2_agentdesign/images/Forage tuned.png}
+    \caption{Tuned Forage Simulation}
+    \label{fig: Forage tuned}
+\end{figure}
+
+Figure~\ref{fig: altruist sim}  shows what happens when the agent plays itself and they are all altruists by default and do not move out of altruist. It can be seen that the agents over contribute to the Common Pool and are left with nothing to forage, this ends the game rather quickly. The simulation result was very similar for when all of the agents were free riders, the game would end in a couple of rounds after a lack of contribution to the common pool which caused impactful disasters.  This proves that being a free rider or altruist is not a rational decision and must be avoided. 
+%the other graph is not really as expected so lets just keep it like this
+
+% \begin{figure}[!htb]
```
Comment (NGovani) [inline 10_team2_agentdesign/index.tex:214]: This needs to be fixed

Comment (NGovani) [inline 10_team2_agentdesign/index.tex]: add figure

**Review by NGovani — DISMISSED** [review]

File: 10_team2_agentdesign/index.tex (hunk 1 above)
Comment (ezgimez) [reply 10_team2_agentdesign/index.tex]: oh yah lol

**Review by ezgimez — CHANGES_REQUESTED** [review]

File: 10_team2_agentdesign/index.tex (hunk 1 above)
Comment (yp717) [reply 10_team2_agentdesign/index.tex]: added :)

**Comment by ezgimez** [comment]
since @yp717  is also one of the main editors, I'm approving this PR (without reading, due to time constraintsssssss).

_DISCLAIMER: I'm not responsible for any grammar/style errors in this PR._

**Review by ezgimez — APPROVED** [review]

**Pull request merged**